prog/vm/nexus.rb

```diff
@@ -301,7 +305,8 @@ def before_run
   end
 
   def write_params_json
-    host.sshable.cmd("sudo -u #{q_vm} tee #{params_path.shellescape}", stdin: vm.params_json(frame["swap_size_bytes"]))
+    host.sshable.cmd("sudo -u #{q_vm} tee #{params_path.shellescape}",
+      stdin: vm.params_json(**frame.slice("swap_size_bytes", "hugepages", "ch_version", "firmware_version").transform_keys(&:to_sym)))
```

Comment:
Could use `transform_keys!` to save an allocation.

Reply:
applied